contracts/ValidatorSet/modules/AccessControl/AccessControl.sol

```diff
@@ -34,12 +34,14 @@ abstract contract AccessControl is IAccessControl, Ownable2StepUpgradeable, Vali
     }
 
     function _addToWhitelist(address account) internal {
-        validators[account].whitelisted = true;
+        if (validators[account].status != ValidatorStatus.None) revert("Previously whitelisted.");
```

Comment:
It would be nice if we make custom errors in the `Errors.sol` in order to keep the codebase identical.

Reply:
I will add a Custom error in the module itself. Better to add common errors only to the Errors.sol file